TeamSnapSDK/SDK/DataTypes/TSDKInvoice.m

```diff
@@ -33,18 +33,40 @@ - (CGFloat)percentPaid {
     return amountPaidFloat/totalFloat;
 }
 
-- (TSDKInvoiceStatus)invoiceStatus {
-    if([[self.status lowercaseString] isEqualToString:[@"open" lowercaseString]]) {
++ (TSDKInvoiceStatus)invoiceStatusForStatusString:(NSString *)statusString {
+    if([[statusString lowercaseString] isEqualToString:[@"open" lowercaseString]]) {
         return TSDKInvoiceStatusOpen;
-    } else if([[self.status lowercaseString] isEqualToString:[@"paid" lowercaseString]]) {
+    } else if([[statusString lowercaseString] isEqualToString:[@"paid" lowercaseString]]) {
         return TSDKInvoiceStatusPaid;
-    } else if([[self.status lowercaseString] isEqualToString:[@"canceled" lowercaseString]]) {
+    } else if([[statusString lowercaseString] isEqualToString:[@"canceled" lowercaseString]]) {
         return TSDKInvoiceStatusCanceled;
     } else {
         return TSDKInvoiceStatusUnknown;
     }
 }
 
++(NSString *_Nonnull)invoiceStatusStringForStatus:(TSDKInvoiceStatus)status {
+    switch (status) {
+        case TSDKInvoiceStatusOpen:
+            return @"open";
+            break;
+        case TSDKInvoiceStatusPaid:
+            return @"paid";
+            break;
+        case TSDKInvoiceStatusCanceled:
+            return @"canceled";
+            break;
+        default:
```

Comment:
If this switch is exhaustive, we shouldn't need a default case

Reply:
It wasn't exhaustive, but it should have been to make it future proof! Fixed.